fc_util/src/lib.rs

```diff
@@ -199,6 +198,12 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_local_time_display_check_month() {
```

Comment:
The scope of this test is too small to warrant a separate test function. You can just add it to `test_local_time_display` above as it's the *display* functionality you're testing.